faiss-sys/src/bindings.rs

```diff
@@ -1,4 +1,4 @@
-/* automatically generated by rust-bindgen 0.71.1 */
+/* automatically generated by rust-bindgen 0.72.0 */
```

Comment:
So I generated the bindings locally, and I got a different bindings.rs file, but the only real differences I could find were in the order of declarations, the API was seemingly the same. I suspect that `rust-bindgen` doesn't guarantee the exact same generated file among different machines. Which is to say that it seems to be in order.